src/applications/personalization/profile360/components/PhoneSection.jsx

```diff
@@ -4,36 +4,20 @@ import PhoneNumberWidget from 'us-forms-system/lib/js/review/PhoneNumberWidget';
 import Vet360ProfileField from '../containers/Vet360ProfileField';
 import PhoneEditModal from './PhoneEditModal';
 
-function renderContent({ data: phoneData }) {
+function PhoneView({ data: phoneData }) {
   const phoneNumber = <PhoneNumberWidget value={[phoneData.areaCode, phoneData.phoneNumber].join('')}/>;
   const countryCode = phoneData.countryCode && <span>+ {phoneData.countryCode}</span>;
   const extension = phoneData.extension && <span>x{phoneData.extension}</span>;
   return <div>{countryCode} {phoneNumber} {extension}</div>;
 }
 
-function renderEditModal({ data: phoneData, title, field, transactionRequest, clearErrors, onChange, onDelete, onSubmit, onCancel }) {
-  return (
-    <PhoneEditModal
-      title={title.toLowerCase()}
-      field={field}
-      error={transactionRequest && transactionRequest.error}
-      clearErrors={clearErrors}
-      onChange={onChange}
-      phoneData={phoneData}
-      onSubmit={onSubmit}
-      isLoading={transactionRequest && transactionRequest.isPending}
-      onCancel={onCancel}
-      onDelete={onDelete}/>
-  );
-}
-
 export default function Vet360Phone({ title, fieldName, analyticsSectionName }) {
```

Comment:
All edit-modals follow the same API as the corresponding content and Vet360ProfileField, so we don't need a middle function that maps props to the edit-modal. The only thing to know here is that the transactionRequest is now exposed to the edit-modal, and that phoneData/addressData/emailData are all now referred to as just data, `this.props.data`.